packages/rn-tester/js/examples/Filter/FilterExample.js

```diff
@@ -253,3 +281,35 @@ exports.examples = [
     },
   },
 ] as Array<RNTesterModuleExample>;
+
+const AnimatedBlurExample = () => {
+  const animatedValue = React.useRef(new Animated.Value(0)).current;
+  const [isBlurred, setIsBlurred] = React.useState(false);
+
+  const onPress = () => {
+    Animated.timing(animatedValue, {
+      toValue: isBlurred ? 0 : 20,
+      duration: 1000,
+      useNativeDriver: false,
```

Comment:
For native driver support we need to merge this - https://github.com/facebook/react-native/pull/52920